datasets/interpress_news_category_tr_lite/README.md

```diff
@@ -0,0 +1,150 @@
+}
+```
+
+### Data Fields
+
+- **category** : Indicates to which category the news text belongs.
+(Such as "kültürsanat", "ekonomi", "siyaset", "eğitim", "dünya", "spor", "teknoloji", "magazin", "sağlık", "gündem")
```

Comment:
```suggestion
(Such as "kültürsanat" (0), "ekonomi" (1), "siyaset" (2), "eğitim" (3), "dünya" (4), "spor" (5), "teknoloji" (6), "magazin" (7), "sağlık" (8), "gündem" (9))
```
we can show the mapping between the label ids (integers) and the label names (strings)